pandas/tests/io/test_excel.py

```diff
@@ -36,6 +36,20 @@
 _mixed_frame['foo'] = 'bar'
 
 
+@contextlib.contextmanager
+def ignore_xlrd_time_clock_warning():
+    """
+    Context manager to ignore warnings raised by the xlrd library,
```

Comment:
hmm, is this *still* present in 1.0.0 which is now our min version?

Reply:
it happens in `1.1.0` (the latest), which is the version I have installed, so I think these warning ignores are needed for now